web-app/HeimScripts/heatmap/run.R

```diff
@@ -104,6 +106,30 @@ buildFields <- function(df){
   return(df)
 }
 
+writeDataForZip <- function(df, patientIDs) {
+  pidCols    <- as.character(patientIDs)
+  t          <- df
+  t[pidCols] <- lapply(t[pidCols], function(v) { (v - df$MEAN) / df$SD })
```

Comment:
Why calculate z-scores here? Shouldn't the result from the z-score calculation earlier in the script be used? Piotr is working now on using z-score values for heatmap (instead of log2 values)